# Lista de estaciones de carga y descarga (2D - [x, y])
float32[] stations

# Lista de estanterías (2D - [x, y])
float32[] shelves

# Lista de muros (2D - [x, y])
float32[] walls

# Posición actual del robot (2D - [x, y])
float32[] robot_position